src/ansiblelint/constants.py

```diff
@@ -126,6 +126,36 @@ def main():
     "pre_tasks",
     "post_tasks",
 ]
+PLAYBOOK_ROLE_KEYWORDS = [
```

Comment:
I got derived this list of playbook role keywords from here https://docs.ansible.com/ansible/latest/reference_appendices/playbooks_keywords.html#role.